textdb/textdb-dataflow/src/main/java/com/google/re2j/RE2J_LICENSE.txt

```diff
@@ -0,0 +1,32 @@
+This is a work derived from Russ Cox's RE2 in Go, whose license
```

Comment:
Yes this is three-clause BSD license. I think the second clause applies to us, since we are not directly using its source code. "Redistributions in binary form must reproduce the above copyright notice". So I feel we should include this license file.